service/controller/v15/version_bundle.go

```diff
@@ -7,6 +7,11 @@ import (
 func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
+			{
+				Component:   "aws-operator",
+				Description: "Enabled termination protection in CloudFormation stacks.",
```

Comment:
`for CloudFormation` is a bit nicer. Although not a big deal.

Reply:
My eternal fight with English prepositions